toutvcli/app.py

```diff
@@ -222,6 +222,8 @@ def _build_argparser(self):
         pf.add_argument('-q', '--quality', action='store',
                         default=App.QUALITY_AVG, choices=quality_choices,
                         help='Video quality (default: {})'.format(App.QUALITY_AVG))
+        pf.add_argument('-v', '--verbose', action='store_true',
```

Comment:
Do we need to have a `--verbose` for each command?  If we only have one in the top-level parser, it forces the user to put the `-v/--verbose` between `toutv` and the subcommand, but is it a problem?  It's a flag that affects the whole program, not just the subcommand, so I think it would make sense.